simplipy/api.py

```diff
@@ -236,6 +241,26 @@ async def _async_api_request(
 
         return data
 
+    async def _async_media_request(
+        self, url: str
+    ) -> bytes | None:
+        """Fetch a media file.
+
+        Args:
+            url: An absolute url for the media file.
+
+        Returns:
+            The raw bytes of the media file
+        """
+        async with self.session.request(
+            "get", url, headers={
+                "User-Agent": DEFAULT_USER_AGENT,
+                "Authorization": f"Bearer {self.access_token}"
```

Comment:
What happens when this token busts?

Reply:
I believe it does the same thing as `_async_api_request()` ... does.  Its being called via backoff just like api requests and the backoff configuration is handling retries and token refreshes based on response status.  At least that's what I think its doing.

Reply:
This function is not called directly, it is called by backoff()